builtin/logical/pki/path_config_cluster.go

```diff
@@ -158,14 +158,16 @@ func (b *backend) pathWriteCluster(ctx context.Context, req *logical.Request, da
 
 		// This field is required by ACME, if ever we allow un-setting in the
 		// future, this code will need to verify that ACME is not enabled.
-		if !govalidator.IsURL(cfg.Path) {
+		u, err := url.Parse(cfg.Path)
+		if err != nil || u.Scheme == "" || u.Host == "" {
```

Comment:
Just a note: I think ACME paths in particular could be validated a touch more but this becomes difficult for reasons I'll attempt to document in a follow-up issue. But for now, this is perfectly fine.